x-pack/plugin/sql/src/main/bin/elasticsearch-sql-cli

```diff
@@ -4,6 +4,8 @@
 # or more contributor license agreements. Licensed under the Elastic License;
 # you may not use this file except in compliance with the Elastic License.
 
+CLI_PARAMETERS="$@"
```

Comment:
Please ping the core-infra folks as well. My preference would be for `CLI_PARAMETERS` to follow CLI_JAR declaration to maintain the structure of the script.

Reply:
`elasticsearch-env` appends to `"$@"`, so it has to be before the `source .../elasticsearch-env` line. This is where the bug comes from.